lib/models/users.js

```diff
@@ -19,18 +20,44 @@ module.exports = function (sequelize, DataTypes) {
             },
             email: {
                 type: DataTypes.STRING(32),
-                allowNull: false
-            },
-            createdAt: {
-                field: 'created_at',
-                type: DataTypes.DATE,
                 allowNull: false,
-                defaultValue: sequelize.literal('CURRENT_TIMESTAMP')
+                unique: 'email'
+            },
+            mobileNo: {
+                field: '',
```

Comment:
any reason to leave field: '' ?

or should we do field: 'mobile_no',?

Reply:
I forgot to add it but now i have added it.